client/src/Components/TopCryptos.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState, useEffect } from "react";
+function TopCryptos() {
+  const [topCryptos, setTopCryptos] = useState([]);
+
+  useEffect(() => {
+    fetch("https://api.coingecko.com/api/v3/search/trending")
+      .then((response) => response.json())
+      .then((data) => setTopCryptos(data.coins))
+      .catch((error) => console.error(error));
+  }, []);
+  console.log(topCryptos);
+  return (
+    <div className="overflow-x-auto">
+      <table className="table" style={{ width: "1400px" }}>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Price</th>
+            <th>Rank</th>
+          </tr>
+        </thead>
+        <tbody>
+          {topCryptos.map((crypto) => {
+            return (
+              <tr>

```

Comment:
could you change this to `<tr key={crypto.id}>` ?
it seems like react requires you to have a key attribute